packages/storybook-a11y-test/src/index.ts

```diff
@@ -139,7 +168,7 @@ export const testPages = async ({
   iframePath,
   storyIds = [],
   concurrentCount = os.cpus().length,
-  timeout = 3000,
+  timeout = 30000,
```

Comment:
Having a 30 second timeout per page means this is going to run forever when there's some weird bug that stops it from resolving. Perhaps a bump to 5000 if 3000 isn't enough?

Reply:
This was a fix! Initially this was set to match Puppeteer's default, but a zero was missing. I agree 30s is a lot, tho.

Reply:
Same thought here. 30s seems like a lot, especially because we run these in parallel.